crm_department/__init__.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
```

Comment:
They are in the maintainer-tools templates, and it's a relief to avoid such big headers, so please change them.